lib/alephant/lookup/lookup_cache.rb

```diff
@@ -0,0 +1,67 @@
+require 'dalli-elasticache'
+require 'alephant/logger'
+
+module Alephant
+  module Lookup
+    class LookupCache
+      include Logger
+
+      DEFAULT_TTL  = 5
+
+      def initialize(config)
+        @config = config || {}
```

Comment:
@samfrench Is there a reason you set the default value inside the method rather than in the method signature? e.g. `initialize(config={})`

There isn't a wrong or right answer per se. The way it is at the moment means that a config *must* be provided when creating a new instance of the class, where as you can make the class a little less fraile by having the argument default value defined within the signature.